### ISSUE TITLE
Crashed when using the library in an "Add-to-App" rather than in a flutter project.

### ISSUE THREAD

**Issue by ksti** [Sh1d0w/multi_image_picker#315]
**Bug Report**

`multi_image_picker` version: 
```
multi_image_picker:
    git:
      url: https://github.com/Sh1d0w/multi_image_picker.git
      ref: 4.6.0-rc.1
```

Is it happening on Android: 

Is it happening on iOS: Yes

```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel master, v1.12.5-pre.14, on Mac OS X 10.15.1 19B88, locale
    zh-Hans-CN)
 
[!] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor
      --android-licenses
[✓] Xcode - develop for iOS and macOS (Xcode 11.1)
[✓] Android Studio (version 3.4)
[✓] VS Code (version 1.40.1)
[✓] Connected device (1 available)

! Doctor found issues in 1 category.
```

**Describe the bug**

Crashed when using the library in an "Add-to-App" rather than in a flutter project.
I have an existing ios app and have added Flutter as Flutter module to it. When open a flutter page it will be crashed at `SwiftMultiImagePickerPlugin.swift` line 45:

```
let controller : FlutterViewController = app.delegate!.window!!.rootViewController as! FlutterViewController;
```

For the "Add-to-App" the `app.delegate!.window!!.rootViewController` is not a `FlutterViewController `, and it will crash.

**Comment by Sh1d0w** [comment]
That's interesting, I haven't thought of that possibility.

Shouldn't your app inherit from flutter view controller as described [here](https://github.com/flutter/flutter/wiki/Add-Flutter-to-existing-apps#write-code-to-use-flutterviewcontroller-from-your-host-app) ?

**Comment by ksti** [comment]
Well, it is possible.
Actually my `AppDelegate` is inherited from `FlutterAppDelegate ` and have a flutterEngine.
```
@property (nonatomic,strong) FlutterEngine *flutterEngine;
```
Maybe you can try app.delegate.flutterEngine.viewController
I will try it soon.

**Comment by Sh1d0w** [comment]
Actually I don't know why I've put that force unwrap there, probably it will be safe to type annotate it with UIViewController or whatever controller FlutterViewController extends?

Feel free to open a PR with a fix if you wish 👍

**Comment by ksti** [comment]
I opened a [PR](https://github.com/Sh1d0w/multi_image_picker/pull/316)

Thank you for your quickly replying and suggestion

**Comment by Sh1d0w** [comment]
Thanks! This is now released in version 4.6.0-rc.2
